packages/terra-clinical-item-view/src/ItemView.jsx

```diff
@@ -106,40 +108,49 @@ const defaultEmphasisContentClassesFromIndexes = (rowIndex, rowCount) => {
     contentSize = 'content-secondary-size';
   }
 
-  if (rowCount === 2 && rowIndex === 1) {
-    contentColor = 'content-secondary-color';
-  } else if (rowIndex >= 2) {
+  if (rowIndex >= 2 || (rowCount === 2 && rowIndex === 1)) {
     contentColor = 'content-secondary-color';
   }
 
   return [contentSize, contentColor];
 };
 
-const startEmphasisContentClassesFromIndexes = (rowIndex, rowCount, contentIndex) => {
-  if (contentIndex === 1) {
+const startEmphasisContentClassesFromIndexes = (rowIndex, rowCount, columnIndex) => {
+  if (columnIndex > 0 || rowIndex >= 2) {
     return ['content-secondary-size', 'content-secondary-color'];
   }
 
   return defaultEmphasisContentClassesFromIndexes(rowIndex, rowCount);
 };
 
-const classesForContent = (rowIndex, rowCount, contentIndex, emphasis) => {
+const classesForContent = (rowIndex, rowCount, columnIndex, emphasis) => {
   let classes;
+
   if (emphasis === TextEmphasisTypes.START) {
-    classes = startEmphasisContentClassesFromIndexes(rowIndex, rowCount, contentIndex);
+    classes = startEmphasisContentClassesFromIndexes(rowIndex, rowCount, columnIndex);
   } else {
     classes = defaultEmphasisContentClassesFromIndexes(rowIndex, rowCount);
   }
+
   return ['content'].concat(classes);
 };
 
-const renderRow = (row, rowIndex, rowCount, emphasis) => {
-  const rowKey = rowIndex;
+const renderColumn = (displayGroup, displayGroupIndex, emphasis) => {
+  const columnKey = displayGroupIndex;
```

Comment:
nit: do we need to assign `columnKey` here or can `displayGroupIndex` be used directly?

Reply:
We technically could use displayGroupIndex but I've been seeing in this file that typically it's set to a key variable first instead of being directly passed in. I'd probably keep this as is unless we go change the others.

Reply:
Makes sense, thanks for clarifying